pkg/scalers/aws_cloudwatch_scaler.go

```diff
@@ -45,8 +45,9 @@ type awsCloudwatchMetadata struct {
 	MetricStatPeriod     int64  `keda:"name=metricStatPeriod,     order=triggerMetadata, default=300"`
 	MetricEndTimeOffset  int64  `keda:"name=metricEndTimeOffset,  order=triggerMetadata, default=0"`
 
-	AwsRegion   string `keda:"name=awsRegion,   order=triggerMetadata;authParams"`
-	AwsEndpoint string `keda:"name=awsEndpoint, order=triggerMetadata, optional"`
+	AwsAccountID string `keda:"name=awsAccountId, order=triggerMetadata, optional"`
+	AwsRegion    string `keda:"name=awsRegion,   order=triggerMetadata;authParams"`
+	AwsEndpoint  string `keda:"name=awsEndpoint, order=triggerMetadata, optional"`
```

Comment:
minor nit regarding alignment
```suggestion
	AwsAccountID string `keda:"name=awsAccountId, order=triggerMetadata, optional"`
	AwsRegion    string `keda:"name=awsRegion,    order=triggerMetadata;authParams"`
	AwsEndpoint  string `keda:"name=awsEndpoint,  order=triggerMetadata, optional"`
```